pkg/maglev/maglev.go

```diff
@@ -28,16 +28,22 @@ const (
 
 	// seed=$(head -c12 /dev/urandom | base64 -w0)
 	DefaultHashSeed = "JLfvgnHc2kaSUFaI"
+
+	maxBackends = 5000
```

Comment:
I'd suggest to make it user configurable.

Reply:
Done